packages/shell-api/src/database.spec.ts

```diff
@@ -20,6 +20,10 @@ import ChangeStreamCursor from './change-stream-cursor';
 import { CommonErrors, MongoshDeprecatedError, MongoshInvalidInputError, MongoshRuntimeError, MongoshUnimplementedError } from '@mongosh/errors';
 chai.use(sinonChai);
 
+type ShellApiErrorsWrapper = Error & {
+  codeName: string
+};
```

Comment:
If we do need to do this (why?) then can we share a single type with this name across the `shell-api` package, instead of multiple slightly divergent ones?

Reply:
Removed all ShellApiErrorsWrappers.